plugin-server/src/main/ingestion-queues/session-recording-v2/consumer.ts

```diff
@@ -111,10 +118,14 @@ export class SessionRecordingIngester {
 
         this.kafkaParser = new KafkaMessageParser()
 
-        this.redisPool = createRedisPool(this.config, 'session-recording')
+        this.redisPool = createRedisPool(this.hub, 'session-recording')
 
         const teamService = new TeamService(postgres)
 
+        this.eventIngestionRestrictionManager = new EventIngestionRestrictionManager(this.hub, {
```

Comment:
was going to suggest injecting this when I saw the change from config to hub in the constructor... amazed that hasn't caused more disruption in test setup 🤯

Reply:
Benefits of having fresh code 😅